addons/knobs/README.md

```diff
@@ -307,8 +307,37 @@ const groupId = 'GROUP-ID1';
 const value = select(label, options, defaultValue, groupId);
 ```
 
-> You can also provide options as an array like this: `['red', 'blue', 'yellow']`.
+Options can also be an array:
 
+```js
+import { select } from '@storybook/addon-knobs';
+const label = 'Cats';
+const options = ['linus', 'eleanor', 'lover']
```

Comment:
Shouldn't `options` be an array of Objects that correspond with `arrayOfObjects`? `defaultValue` is `arrayOfObjects[0]`, but that object doesn't match anything within `options`. This appears *broken*, but perhaps I am not understanding something. 

Now, looking at https://github.com/storybookjs/storybook/pull/7957/files#diff-1b10673d7ef02f5f40b685bb83416afeR92, this appears to be configured correctly, where the `arrayOfObjects[0]` can be matched against something within `arrayOfObjects`.

Reply:
yup you got it!